zproject/backends.py

```diff
@@ -1423,7 +1423,21 @@ def auth_url(self) -> str:
         relay_state = self.put_data_in_redis(data_to_relay)
 
         return auth.login(return_to=relay_state)
-
+    
+    def _check_entitlements(self, idp, attributes):
+        """
+        Check if we require the presence of any specific eduPersonEntitlement.
+        raise AuthForbidden if the user should not be authenticated, or do nothing
+        to allow the login pipeline to continue.
```

Comment:
Can you extend this docstring to mention this part of the python-social-auth API for SAML backends, with a link to:
https://python-social-auth.readthedocs.io/en/latest/backends/saml.html

That'll be helpful for making less knowledge on the part of developers familiar mostly with other parts to Zulip required to read this :)